### PR TITLE
Fix up libpod.conf man pages and referencese to it.

### PR DESCRIPTION
**Pull request by rhatdan**
Remove podman --config option, since it does not do anything.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
I figured I would open a PR for the less controversial parts of https://github.com/containers/libpod/pull/1406

**Comment by mheon** [comment]
I think we should just implement `--config` - I know this worked originally, but it might have been broken around the original Varlink changes.

**Comment by mheon** [comment]
Just need a `GlobalIsSet("config")` and if it is set, call `NewRuntimeFromConfig` instead of `NewRuntime`

File: docs/libpod.conf.5.md [hunk 1]
```diff
@@ -45,8 +45,24 @@ libpod to manage containers.
 **cni_plugin_dir**=""
   Directories where CNI plugin binaries may be located
 
+**pause_image** = ""
+  Pause container image name for pod pause containers.  When running a pod, we
+  start a `pause` processes in a container to hold open the namespaces associated with the
```
Comment (jamescassell) [inline docs/libpod.conf.5.md]: Singular 'process'?

Comment (rhatdan) [reply docs/libpod.conf.5.md]: FIxed

**Comment by mheon** [comment]
Changes LGTM

**Comment by rhatdan** [comment]
We are getting weird spurious failures.

File: docs/libpod.conf.5.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/libpod.conf.5.md]: My head still spins over "pod pause containers".  Does this work?
"The image name to use for the containers in the pod pause command."  or perhaps "... {running || controlling} the pod pause command."

Comment (rhatdan) [reply docs/libpod.conf.5.md]: Sometimes this is called the infracontainer.  Perhaps we should change the lingo now to clear this up.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=mheon

**Pull request closed without merging**